xgcm/test/test_grid.py

```diff
@@ -784,34 +784,52 @@ def test_boundary_kwarg_same_as_grid_constructor_kwarg():
 
 @pytest.mark.parametrize(
     "metric_axes,metric_name",
-    [("X", "dx_t"), ("Y", "dy_ne"), (["Y", "X"], "dy_n")],
+    [
+        (["Y", "X"], "area_n"),
+        ("X", "dx_t"),
+        ("Y", "dy_ne"),
+        (["Y", "X"], "dy_n"),
+        (["X"], "tracer"),
+    ],
 )
-def test_interp_like(metric_axes, metric_name):
-
-    ds, coords, _ = datasets_grid_metric("C")
-    grid = Grid(ds, coords=coords)
-    grid.set_metrics(metric_axes, metric_name)
-    metric_available = grid._metrics.get(frozenset(metric_axes), None)
-    metric_available = metric_available[0]
-    interp_metric = grid.interp_like(metric_available, ds.u)
-    test_metric = grid.interp(ds[metric_name], metric_axes)
-
-    xr.testing.assert_equal(interp_metric, test_metric)
-    xr.testing.assert_allclose(interp_metric, test_metric)
-
-
+@pytest.mark.parametrize("periodic", [True, False])
 @pytest.mark.parametrize(
-    "var_name,like_name,var_axes",
+    "boundary, boundary_expected",
```

Comment:
I don't think we need `boundary_expected` at all? You can just pass `boundary` to `interp`?

Reply:
There's a test where boundary = 'fill' and the boundary_expected is {"X": "fill", "Y": "fill"}, and we want to make sure that the boundaries are still assigned correctly. There's also the pytest.mark.xfail case where `boundary` and `boundary_expected` are not the same, so for these cases I think it's worth it to have them as separate variables.